app/models/picture.rb

```diff
@@ -0,0 +1,16 @@
+class Picture < ActiveRecord::Base
+
+  belongs_to :user
+  delegate :email, to: :user
+  delegate :name, to: :user
+  delegate :avatar, to: :user
+  delegate :thoughts, to: :comment
+
+  has_many :comments
+
+  has_attached_file :image, :styles => { :large => "750x", :medium => "600x", :thumb => "100x" }, :default_url => "/images/:style/missing.png"
```

Comment:
Metrics/LineLength: Line is too long. [142/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)